File: Schedule.Core/DTO/Studying/OrganizationDto.cs
using System;
using System.Collections.Generic;

namespace Schedule.Core.DTO.Studying
{
    public class OrganizationDto
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public IEnumerable<FacultyDto> Faculties { get; set; }
    }
}

File: Schedule/ClientApp/App/src/app/studying/models/OrganizationDto.ts
import { Guid } from 'guid-typescript/dist/guid';
import { FacultyDto } from './FacultyDto';

export class OrganizationDto {
  id: Guid;
  name: string;
  faculties: FacultyDto[];

  constructor(model: Partial<OrganizationDto>) {
    this.id = model.id;
    this.name = model.name;
    this.faculties = model.faculties;
  }
}
